app/src/main/res/values/strings.xml

```diff
@@ -21,4 +21,12 @@
     <string name="feed_item_location_default">Street Name, Num, City, State, ZipCode</string>
     <string name="feed_item_price_default">$ $ $ $</string>
 
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Nop... this string is part of the default code injected when using a template... please do not inject useless code